test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -338,7 +413,7 @@ export default class PerfCompare extends Component {
             await this.handleGetJenkinsRuns();
 
             // Received a Perffarm Run URL
-        } else {
+        } else if (this.state.buildType === 'Perffarm') {
```

Comment:
brittle comparison?  Const should be defined somewhere?

Reply:
This seems sane enough to me, we set `this.state.buildType` as part of the react UI selector so the values are hardcoded.